### DISCUSSION TITLE
Version 1.4.2/1.5.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#940] [Changelog]
We probably want to release version 1.4.2 (or 1.5.0)

## Bugfixes
- #669 editor-theme3 did not affect devtools middle click feature. (prio-4)
- #925 Most translations don't load. (prio-3)
- #925 My Ocular addon is not localized. (prio-4)
- #925 "Search" placeholder is not localized. (prio-4)
- #928 Project comments are unreadable with dark mode. (prio-3)
- #929 "Show More" button is partially hidden. (prio-4)
- #930 Some addons overflow in small stage. (prio-4)
- #935 Addons sometimes don't load in Firefox. (prio-2)
- #941 /news is not listed as scratchr2 page. (prio-4)
- #944 Warning when posting on Japanese forum. (prio-4)
- #948 Custom colors do not apply to custom blocks. (prio-3)
- #949 Some pages are not listed in dark mode list. (prio-4)
- #950 Warning when site access is off. (prio-2)
- #954 mouse-pos crashes in project view. (prio-2)
- #956 Custom colors do not apply to dropdowns. (prio-3)
- #956 Custom colors do not apply to extension blocks in non-English environment. (prio-4)
- #960 dark-www still loads on editors. (prio-3)
- #960 dark-www does not make cloudmonitor rows dark. (prio-4)
- #969 scratch-messaging links to curators page for activity messages. (prio-4)
- #969 Follower name links in scratch-messaging use onClick. (prio-4)
- #981 dark-www changes variable monitor color. (prio-4)
- #981 dark-www styles in-editor Join Flow. (prio-4)
- #981 dark-www styles list row backgrounds when focused. (prio-4)
- #989 forum-quote-code-beautifier is categorized as editor addon. (prio-4)
- #994 studio-tools buttons don't have text when hovered. (prio-3)
- #995 dark-www does not apply to loading screen. (prio-3)
- #995 dark-www makes search input dark. (prio-4)
- #995 dark-www makes project warnings unreadable. (prio-4)
- Localization updates

## New features
- #879 New addon for onion skins (prio-3)
- #895 New addon for adding profile page banners (prio-4)
- #897 Addons don't load inside frames. (prio-3)
- #899 Show full areas supports loading more in What's Happening (prio-4)
- #964 Add a way to disable thumbnail overwriting. (prio-3)
- #979 Block switching supports more blocks. (prio-4)
- #980 New devtools feature for showing broadcast senders and receivers (prio-3)
- #1002 Add a button to open translation page. (prio-3)
- #1016 Add notification for updates (prio-3)
- #1016 Add "New" tag for addons. (prio-4)

**Comment by ghost** [comment]
BUMP! 1.5.0. is stable!
